bundles/org.openhab.binding.teslapowerwall/src/main/java/org/openhab/binding/teslapowerwall/internal/TeslaPowerwallBindingConstants.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.teslapowerwall.internal;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link TeslaPowerwallBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Paul Smedley - Initial contribution
+ */
+@NonNullByDefault
+public class TeslaPowerwallBindingConstants {
+
+    private static final String BINDING_ID = "teslapowerwall";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID TESLAPOWERWALL_THING = new ThingTypeUID(BINDING_ID, "tesla-powerwall");
+
+    // List of all Channel ids
+    public static final String CHANNEL_TESLAPOWERWALL_GRIDSTATUS = "grid-status";
+    public static final String CHANNEL_TESLAPOWERWALL_GRIDSERVICES = "gridservices";
```

Comment:
```suggestion
    public static final String CHANNEL_TESLAPOWERWALL_GRIDSERVICES = "grid-services";
```

please adapt on all places. (readme, xml etc)

Reply:
Done, also found this wasn't documented in README.md